tekstowo.py

```diff
@@ -0,0 +1,464 @@
+# Necessary imports
+from typing import Tuple, List, Dict
+from datetime import datetime
+from bs4 import BeautifulSoup
+
+import os
+import argparse
+import string
+import time
+import requests
+import langdetect
```

Comment:
It would be proper to sort imports like this: first group is standard library imports and second group are third party imports. Information about necessary third party imports could be written in requirements.txt file.

import argparse
import os
import string
import time
from datetime import datetime
from typing import Dict, List, Tuple

import langdetect
import requests
from bs4 import BeautifulSoup

Reply:
Updated with the usage of isort, requirements.txt added.